### PR TITLE
Energy adjustments

### PR DESCRIPTION
**Pull request by Humonitarian**
## About The Pull Request

Adjust penetration value of rapid crossbow and sawn off cog, adjusts firerate of plasma weapons to better measure up to new ballistic firerates.

## Why It's Good For The Game

Balance.

## Changelog
:cl:
balance: sawn off Cog and rapid crossbow penetration stat fixed
balance: plasma firerates increased, plasma firemode related firerate bugs fixed
/:cl:

### CODE REVIEW

**Comment by assortedbeads** [comment]
They nerfed my baby, literally 1984

**Comment by maikilangiolo** [comment]
> They nerfed my baby, literally 1984

It had 100% more AP that it should have. It had increased AP, even.

**Comment by Humonitarian** [comment]
> > They nerfed my baby, literally 1984
> 
> It had 100% more AP that it should have. It had increased AP, even.

He knows, we've been joking about it for a while

File: code/modules/projectiles/guns/energy/rapidcrossbow.dm [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 	icon = 'icons/obj/guns/energy/poweredcrossbow.dmi'
 	icon_state = "rxb_empty"
 	item_state = "rxb_empty"
 	flags = CONDUCT
-	slot_flags = SLOT_BACK|SLOT_BELT|SLOT_HOLSTER 
+	slot_flags = SLOT_BACK|SLOT_BELT|SLOT_HOLSTER
 	wielded_item_state = "_doble"
 	damage_multiplier = 0.8
-	penetration_multiplier = 0.9
+	penetration_multiplier = -0.1
```
Comment (TheShown911) [inline code/modules/projectiles/guns/energy/rapidcrossbow.dm:11]: The word multiplier implies multiplication, maybe change it to modifier in another PR.

**Review by TheShown911 — APPROVED** [review]

**Pull request merged**